Tests/UnitTests/Purchasing/Purchases/PurchasesConfiguringTests.swift

```diff
@@ -362,6 +362,92 @@ class PurchasesConfiguringTests: BasePurchasesTests {
         expect(self.paymentQueueWrapper.delegate) === self.purchasesOrchestrator
     }
 
+    // MARK: - Custom Entitlement Computation
+    func testCustomEntitlementComputationSkipsFirstDelegateCall() throws {
+        self.systemInfo = MockSystemInfo(finishTransactions: true,
+                                         customEntitlementsComputation: true)
+        self.setupPurchases()
+
+        expect(self.purchasesDelegate.customerInfoReceivedCount).toEventually(equal(0))
+    }
+
+    func testWithoutCustomEntitlementComputationDoesntSkipFirstDelegateCall() throws {
+        self.systemInfo = MockSystemInfo(finishTransactions: true,
+                                         customEntitlementsComputation: false)
+        self.setupPurchases()
+
+        expect(self.purchasesDelegate.customerInfoReceivedCount).toEventually(equal(1))
+    }
+
+    func testConfigureWithCustomEntitlementComputationFatalErrorIfNoAppUserID() throws {
+        self.systemInfo = MockSystemInfo(finishTransactions: true,
+                                         customEntitlementsComputation: true)
+
+        let expectedMessage = Strings.configure.custom_entitlements_computation_enabled_but_no_app_user_id.description
+
+        expectFatalError(expectedMessage: expectedMessage, testcase: {
+            self.setupAnonPurchases()
+        })
+    }
+
+    func testConfigureWithCustomEntitlementComputationNoFatalErrorIfAppUserIDPassedIn() throws {
+        self.systemInfo = MockSystemInfo(finishTransactions: true,
+                                         customEntitlementsComputation: true)
+        expectNoFatalError(testcase: {
```

Comment:
You could use trailing syntax here:
```suggestion
        expectNoFatalError {
```

Reply:
oh nice